deployment/helm/node-feature-discovery/values.yaml

```
@@ -422,6 +422,44 @@ topologyUpdater:
   annotations: {}
   affinity: {}
 
+topologyGC:
+  enable: false
```

In the new setup we want to enable topology-gc by default

```suggestion
  enable: true
```

---

Is it make sense to enable `topologyGC` by default when `topologyUpdater` is disabled by default(as well as creating NRT CRDs)?

---

My bad :) forgot that we only deploy `GC` when `updater` is enabled